cpp/src/parquet/properties.h

```diff
@@ -819,8 +819,7 @@ class PARQUET_EXPORT ArrowWriterProperties {
           coerce_timestamps_unit_(::arrow::TimeUnit::SECOND),
           truncated_timestamps_allowed_(false),
           store_schema_(false),
-          // TODO: At some point we should flip this.
```

Comment:
```
/// This is disabled by default, but will be enabled by default in future.
```

Should we change this in Loc 880?

Reply:
The line 320 in the parquet.rst doc should also be changed.
```
   std::shared_ptr<ArrowWriterProperties> arrow_props = ArrowWriterProperties::Builder()
      .enable_deprecated_int96_timestamps() // default False
      ->store_schema() // default False
      ->enable_compliant_nested_types() // default False
      ->build();

```